packages/core/src/animation/AnimatorStateMachine.ts

```diff
@@ -10,6 +10,9 @@ export class AnimatorStateMachine {
   /** The list of states. */
   readonly states: AnimatorState[] = [];
 
+  /** The state will be played automatically */
+  entryState: AnimatorState;
```

Comment:
I like `defaultState`, means the default play state.

comment must add `.` and not clearly

Reply:
The editor named entryState, i think it should be samed

Reply:
OK! Let me see see editor!